github/Artifact.py

```diff
@@ -0,0 +1,157 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2021 Yixin Guo <yixin.guo@ni.com>                                  #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+
+import re
+
+import requests
+
+import github.GithubObject
+
+
+class Artifact(github.GithubObject.CompletableGithubObject):
+    """
+    This class represents Workflow Run Artifacts.
+    The reference can be found here https://docs.github.com/en/rest/reference/actions#artifacts
+    """
+
+    def __repr__(self):
+        return self.get__repr__({"id": self._id.value, "url": self._url.value})
+
+    def get_file_content(self):
```

Comment:
This is the only method that does not have a test mainly because I didn't know how to go through the GithubObject connection to get the file content and used requests instead.